.github/workflows/workflow.yml

```diff
@@ -19,14 +19,14 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-go@master
         with:
-          go-version: '^1.14.0'
+          go-version: "1.17.7"
       - name: Install dependencies (ubuntu-latest)
         run: |
           sudo apt-get update --fix-missing
           sudo apt-get autoremove
           sudo apt-get autoclean
       - name: Golang code style check (libp2p_node)
-        uses: golangci/golangci-lint-action@v1
+        uses: golangci/golangci-lint-action@v3
```

Comment:
we use @v3 in workflow in [open-aea workflow](https://github.com/valory-xyz/open-aea/blob/d72f99a960ce1f81c2083206e00acf11b83e4aa4/.github/workflows/workflow.yml#L80)